Demo/Common/IntegrationExamples/Components/InstantPaymentComponentAdvancedFlowExample.swift

```diff
@@ -10,7 +10,7 @@ import Adyen
 import AdyenActions
 import AdyenComponents
 
-internal final class InstantPaymentComponentAdvancedFlowExample: InitialDataAdvancedFlowProtocol {
+internal final class InstantPaymentComponentAdvancedFlow: InitialDataAdvancedFlowProtocol {
```

Comment:
Yes I made it shorter because the linter complained about the length - will think about how to have it aligned but still shorter in a subsequent PR :)